lib/services/project-templates-service.ts

```diff
@@ -1,13 +1,99 @@
 ///<reference path="../.d.ts"/>
 "use strict";
+import * as path from "path";
+import * as temp from "temp";
+import * as constants from "../constants";
+temp.track();
 
 export class ProjectTemplatesService implements IProjectTemplatesService {
-	private static NPM_DEFAULT_TEMPLATE_NAME = "tns-template-hello-world";
+	private static RESERVED_TEMPLATE_NAMES: IStringDictionary = {
```

Comment:
The current code is written to be case sensitive for template names. I think it is more user friendly to make it case insensitive but do not insist. Opinions?